### ISSUE TITLE
what does fp8_unet do? 

### ISSUE THREAD

**Issue by MaxTran96** [kijai/ComfyUI-SUPIR#97]
what does fp8_unet do?  Does that save vram if we enable fp8_unet?

**Comment by kijai** [comment]
Yes it saves quite a lot of VRAM, but there's possibly a slight quality hit.
